pegasus/sites.v3/code.org/public/css/page/hour-of-code-styles.scss

```diff
@@ -49,6 +50,30 @@
   }
 }
 
+// 2024 soon-hoc hero banner styles
+.hero-banner-basic.hoc-2024 {
+  background: none; // this overrides the background image from the previous class
```

Comment:
My CSS skills are still not amazing ... this works because `.hero-banner-basic.hoc-2024` is more specific than `.hero-banner-basic` right? Intuitively that makes sense, but CSS specificity rules have not made sense to me a couple times 😅

Reply:
Yep, that's correct! This'll override the existing banner styles on the new 2024 banner while keeping the current AI banner as-is.